logistics_project/apps/malawi/warehouse/runner.py

```diff
@@ -497,10 +497,15 @@ def update_consumption_times(since):
     """
     Update the consumption time_with_data values for any supply point / product pairings
     where the supply point doesn't explicitly supply the product.
+    This function only works against base level data.
     """
     # any consumption value that was touched potentially needs to have its
     # time_needing_data updated
-    consumptions_to_update = CalculatedConsumption.objects.filter(update_date__gte=since)
+    consumptions_to_update = CalculatedConsumption.objects.filter(
+        Q(product__type__base_level=BaseLevel.HSA, supply_point__type__code=SupplyPointCodes.HSA) |
+        Q(product__type__base_level=BaseLevel.FACILITY, supply_point__type__code=SupplyPointCodes.FACILITY),
+        update_date__gte=since
+    )
```

Comment:
@czue it seemed to me like this should only operate on base level data, is that right?

with the previous behavior even if it were to catch and update aggregated data, the aggregated data would normally be fixed automatically when the aggregates run in the same warehouse run.  but it causes a problem when we fix data historically like in this pr in which case the aggregates would not run again for past months.